components/NewPersonView/UnfinishedSubmissions.spec.tsx

```diff
@@ -60,7 +60,7 @@ describe('UnfinishedSubmissions', () => {
     expect(screen.getAllByRole('link').length).toBe(3);
   });
 
-  it('truncates a long list', () => {
```

Comment:
We don't truncate a list now as we already return a paginated list of what we want. Reason for removal of .slice in UnfinishedSubmissions component.